corehq/apps/sms/management/commands/fix_preferred_number.py

```diff
@@ -0,0 +1,81 @@
+from django.core.management.base import BaseCommand, CommandError
+from optparse import make_option
+from corehq.apps.sms.mixin import VerifiedNumber
+from corehq.apps.users.models import CouchUser
+from corehq.apps.reminders.util import (get_verified_number_for_recipient_old,
+    get_verified_number_for_recipient)
+
+
+class Command(BaseCommand):
+    args = ""
+    help = "Update default phone number based on new methodology."
+    option_list = BaseCommand.option_list + (
+        make_option("--fix",
+                    action="store_true",
+                    dest="fix",
+                    default=False,
+                    help="Include this option to automatically update the default phone numbers."),
+    )
+
+    def phones_are_strings(self, user):
+        for phone_number in user.phone_numbers:
+            if not isinstance(phone_number, basestring):
+                return False
+        return True
+
+    def handle(self, *args, **options):
+        # This is ok, there's only like 50k of these and we're only querying ids
+        vns = VerifiedNumber.view(
+            'sms/verified_number_by_owner_id',
+            include_docs=False
+        ).all()
+
+        # Convert to a dict of {owner id: count of total numbers}
+        owners = {}
+        for vn in vns:
+            owner_id = vn['key']
+            if owner_id in owners:
+                owners[owner_id] += 1
+            else:
+                owners[owner_id] = 1
+
+        # Convert to a list of owner ids that have more than one VerifiedNumber
+        # (excluding pending numbers)
+        mult_list = []
+        for owner_id, count in owners.iteritems():
+            if count > 1:
+                owner_vns = VerifiedNumber.view(
+                    'sms/verified_number_by_owner_id',
+                    key=owner_id,
+                    include_docs=True
+                ).all()
```

Comment:
nope, there's just like 2-3 numbers per user, and there's < 50 users we need to run this for